src/main/java/functions/AnalyseCommand.java

```diff
@@ -0,0 +1,71 @@
+package functions;
+
+public class AnalyseCommand {
+
+    protected String commandType;
+    protected String commandDescription;
+    protected String commandTime;
+
+    public AnalyseCommand(String input) {
+        setCommandType(input);
+        setCommandDescription(input);
+        setCommandTime(input);
+
```

Comment:
I think this blank line can be removed to make the code more elegant

Reply:
Resolved. Thank you!